app/dashboard/tests/test_dashboard_helpers.py

```diff
@@ -50,3 +55,102 @@ def test_amount(self):
+                    "created_on": "2019-02-07T15:18:38.678328Z",
+                    "accepted_on": "2019-02-07T15:18:38.678320Z",
+                    "fulfiller_github_url": ""
+                }
+            ],
+            "interested": [],
+            "is_open":True,
+            "expires_date": "2020-02-07T14:38:22Z",
+            "activities": [],
+            "keywords": "",
+            "current_bounty":True,
+            "value_in_eth": "1000000000000000.00",
+            "token_value_in_usdt": "104.60",
+            "value_in_usdt_now": "0.00",
+            "value_in_usdt": "0.00",
+            "status": "done",
+            "now": "2019-02-08T12:38:36.208491Z",
+            "avatar_url": "https://gitcoin.co/dynamic/avatar/danlipert",
+            "value_true": "0.00",
+            "issue_description": "adsgadfh",
+            "network": "rinkeby",
+            "org_name": "danlipert",
+            "pk": 8564,
+            "issue_description_text": "adsgadfh",
+            "standard_bounties_id": 1725,
+            "web3_type": "bounties_network",
+            "can_submit_after_expiration_date":True,
+            "github_issue_number": 5,
+            "github_org_name": "danlipert",
+            "github_repo_name": "gitcoin-test",
+            "idx_status": "done",
+            "token_value_time_peg": "2019-02-07T14:38:22Z",
+            "fulfillment_accepted_on": "2019-02-07T15:18:38.678320Z",
+            "fulfillment_submitted_on": "2019-02-07T15:18:38.678328Z",
+            "fulfillment_started_on": 'null',
+            "canceled_on": 'null',
+            "canceled_bounty_reason": "",
+            "issuer": "null",
+            "action_urls": {
+                "fulfill": "/issue/fulfill?pk=8564&network=rinkeby",
+                "increase": "/issue/increase?pk=8564&network=rinkeby",
+                "accept": "/issue/accept?pk=8564&network=rinkeby",
+                "cancel": "/issue/cancel?pk=8564&network=rinkeby",
+                "payout": "/issue/payout?pk=8564&network=rinkeby",
+                "contribute": "/issue/contribute?pk=8564&network=rinkeby",
+                "advanced_payout": "/issue/advanced_payout?pk=8564&network=rinkeby",
+                "social_contribution": "/issue/social_contribution?pk=8564&network=rinkeby",
+                "invoice": "/issue/invoice?pk=8564&network=rinkeby"
+            },
+            "project_type": "traditional",
+            "permission_type": "approval",
+            "attached_job_description": "",
+            "needs_review":False,
+            "github_issue_state": "closed",
+            "is_issue_closed":True,
+            "additional_funding_summary": {},
+            "funding_organisation": "",
+            "paid": [
+                "danlipert"
+            ],
+            "admin_override_suspend_auto_approval":True,
+            "reserved_for_user_handle": "",
+            "is_featured":True
+        },
+        process_bounty_details(bounty_payload)
```

Comment:
@iamonuwa try calling `create_new_bounty` here, either directly or indirectly

Reply:
`create_new_bounty` is not called directly in the code. Its called through `process_bounty_details`

Reply:
The additional parameters for `create_new_bounty` is generated inside the `process_bounty_details`

Reply:
@danlipert can you confirm?